src/diffusers/utils/__init__.py

```diff
@@ -55,6 +55,8 @@
     USE_TORCH,
     DummyObject,
     OptionalDependencyNotAvailable,
+    _is_torch_fp16_available,
+    _is_torch_fp64_available,
```

Comment:
Importing torch would affect the lazy import speed. I think we can move these functions into `testing_utils` and remove importing them in `utils`.